keyboards/takashicompany/worm_eaten/config.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2023 takashicompany (@takashicompany)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#ifdef INTEGRATED
+	#define MATRIX_ROWS 4
+	#define MATRIX_COLS 10
+	#define MATRIX_COL_PINS { F4, F5, F6, F7, B1, B3, B2, B6, B5, B4 }
+#else
+	#define MATRIX_ROWS 4 * 2
+	#define MATRIX_COLS 5
+	#define MATRIX_COL_PINS { F4, F5, F6, F7, B1 }
+#endif
```

Comment:
This should be implemented as 2 sub revisions and not with ifdef logic, for example

```
keyboards/takashicompany/worm_eaten/keymaps/default/keymap.c
keyboards/takashicompany/worm_eaten/integrated/config.h
keyboards/takashicompany/worm_eaten/split/config.h
```

Reply:
@zvecr 

Thank you very much.
I will make the changes as intended.

One question, is it possible to make the suggested folder hierarchy and build the default keymap using config.h in split or integrated?
I am wondering if it is possible to build the keymap folder without including it in the split or integrated folders, as I have no experience with this.

Reply:
The keymap folder, as zvecr listed it will compile for both of the "integrated" and "split" revisions.  Similar can be seen with boards like the planck, ergodox_ez, corne/crkbd, and many others.  

When compiling a revision, the build process does look at higher and lower level folders for the keymap folders.

Reply:
Thanks for the clear explanation.
I see, so the keymap folder does not necessarily have to be at the bottom level. I learned a lot.

I have made the correction and would appreciate your review.